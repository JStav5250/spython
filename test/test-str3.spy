str x = "remember plt 2024"
print(x)
