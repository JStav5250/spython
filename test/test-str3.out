remember plt 2024
